CMSgov/design-system
# [WNMGDS-2530] Sketch plugin: Generate text styles from tokens

## Summary

Building on https://github.com/CMSgov/design-system/pull/2736, this adds support for text-related tokens. The new code...
- Goes through each component's set of tokens
    - finding text tokens
    - and grouping them by the state that they represent (like "success" or "disabled")
- Converts those grouped text properties into Sketch's _shared text styles_
    - with a name that represents that component and state
    - and filling in any missing values with those from a "base" text style that [matches what our html elements would inherit](https://github.com/CMSgov/design-system/blob/pwolfert/sketch-tokens-update-text-styles/packages/design-system/src/styles/_reset.scss#L35-L38)
 - Resulting in a set of named groups of named text styles that can be used in individual components
    - And one of those groups is `typography`, which [corresponds to this set of typography tokens](https://github.com/CMSgov/design-system/blob/pwolfert/sketch-tokens-update-text-styles/packages/design-system-tokens/src/themes/core-component.ts#L437-L446)


## How to test

See previous pull request for instructions on running the plugin. New text styles will be in the text styles section of Sketch.
